sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -1838,7 +1838,7 @@ public virtual AnalyzeHealthcareEntitiesOperation AnalyzeHealthcareEntities(Wait
         /// <exception cref="NotSupportedException">This method is only supported in service API version v3.1 and newer.</exception>
         /// <exception cref="RequestFailedException">Service returned a non-success
         /// status code.</exception>
-        public virtual AnalyzeHealthcareEntitiesOperation AnalyzeHealthcareEntities(WaitUntil waitUntil, IEnumerable<TextDocumentInput> documents, AnalyzeHealthcareEntitiesOptions options, CancellationToken cancellationToken = default)
+        public virtual AnalyzeHealthcareEntitiesOperation AnalyzeHealthcareEntities(WaitUntil waitUntil, IEnumerable<TextDocumentInput> documents, AnalyzeHealthcareEntitiesOptions options = default, CancellationToken cancellationToken = default)
```

Comment:
This library is GAed.

Its corresponding async method has required `AnalyzeHealthcareEntitiesOptions options`.

Another solution is we add suppression to the method.

Reply:
I thought there was an autorest property to turn this off. It was used for CLU. /cc @m-nash

Reply:
@heaths what you want to off? Are you saying `AnalyzeHealthcareEntitiesOptions options` should keep required?

Reply:
@tg-msft this isn't a breaking change, but do you see any reasons we shouldn't take this particular change?

Reply:
No, I think this is fine.  It's not a binary or source break and won't cause any confusion for existing users.

Reply:
@heaths We turned off [optional parameters with real values](https://github.com/Azure/autorest.csharp/pull/3479), since these are problematic to version.  `default` and `null` are still allowed.